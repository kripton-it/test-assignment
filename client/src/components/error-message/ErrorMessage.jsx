import React from "react";

import * as Styles from './styles';

export class ErrorMessage extends React.Component {
  render() {
    return (
      <Styles.ErrorContainer>
        <Styles.ErrorText>Sorry, this page is broken</Styles.ErrorText>
      </Styles.ErrorContainer>
    );
  }
}

export default ErrorMessage;