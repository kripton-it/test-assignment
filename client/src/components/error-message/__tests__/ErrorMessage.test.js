import React from "react";
import { shallow } from "enzyme";
import toJson from 'enzyme-to-json';

import ErrorMessage from "../ErrorMessage";

it("expect to render ErrorMessage component", () => {
  const wrapper = shallow(<ErrorMessage />);
  expect(toJson(wrapper)).toMatchSnapshot();
});
