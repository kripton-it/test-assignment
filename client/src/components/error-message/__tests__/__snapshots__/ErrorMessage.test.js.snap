// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render ErrorMessage component 1`] = `
<ErrorContainer>
  <ErrorText>
    Sorry, this page is broken
  </ErrorText>
</ErrorContainer>
`;
